packages/p2p/src/components/orders/chat/chat-header.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { observer } from 'mobx-react-lite';
+import { useStores } from 'Stores';
+import ExtendedOrderDetails from 'Utils/orders';
+import { generateHexColourFromNickname, getShortNickname } from 'Utils/string';
+
+const ChatHeader = observer(() => {
+    const { general_store, sendbird_store } = useStores();
+    const { other_user_details } = general_store.order_information;
+    const icon_background_colour = generateHexColourFromNickname(other_user_details.name);
+    const short_nickname = getShortNickname(other_user_details.name);
+
+    return (
+        <div className='order-chat__header'>
+            {isMobile() && (
```

Comment:
We have an `is_mobile` from `general_store.props`, just so we keep it consistent, which would be better?

Reply:
They calculate the same thing other than that `is_mobile` from UI store isn't a function, so in other words P2P would have to rely on the consumer to ensure that value is correct. I think we can get rid of passing that prop and use `isMobile` wherever we'd need it. 

While `is_mobile` from Deriv.app's UI store will most likely always reflect the correct value, I'm not sure whether that would be the same for Binary Static.